content/community/sig/imaging/index.md

```diff
@@ -0,0 +1,124 @@
+---
+description: "If you need to extract quantitative information from images and want to make your workflows reusable, this community is for you." 
+autotoc: true
+title: "iMAGE-NET – Image Analysis Network in Galaxy"
```

Comment:
let's discuss this in the broader group

Reply:
Are we changing it to: Galaxy Image Network (GIN)🍸?

Reply:
I'm collecting here some arguments I've heard against it over lunch:
- Network can be confused with neural network for image analysis
- Image analysis doesn't include e.g. image processing
- To be consistent with the other communities, it would be good to include the word "community" there